Apps/WebClient/src/ClientApp/src/views/profile.vue

```diff
@@ -543,15 +547,15 @@ export default class ProfileView extends Vue {
                                                             : false
                                                     "
                                                 />
-                                                <b-button
+                                                <BaseButtonComponent
                                                     v-if="
                                                         !emailVerified &&
                                                         !isEmailEditable &&
                                                         email
                                                     "
                                                     id="resendEmail"
                                                     data-testid="resendEmailBtn"
-                                                    variant="outline-primary"
+                                                    button-type="secondary"
```

Comment:
if those values are going to match bootstrap variants (primary, secondary, etc) maybe we could you the same property name? variant="secondary" instead of button-type="secondary"

Reply:
Thanks, I think I will update it to "variant" after thinking about it some more.  We currently only support "primary" and "secondary" in the design system, but it will probably be updated to include at least "warning" and "danger" in the future, so it may be better to stick with something familiar.